src/main/resources/org/sonar/plugins/findbugs/rules-findbugs.xml

```diff
@@ -88,9 +88,9 @@ While ScheduledThreadPoolExecutor inherits from ThreadPoolExecutor, a few of the
     <description>&lt;p&gt;This code constructs an HTTP Cookie using an untrusted HTTP parameter. If this cookie is added to an HTTP response, it will allow a HTTP response splitting
 vulnerability. See &lt;a href="http://en.wikipedia.org/wiki/HTTP_response_splitting"&gt;http://en.wikipedia.org/wiki/HTTP_response_splitting&lt;/a&gt;
 for more information.&lt;/p&gt;
-&lt;p&gt;FindBugs looks only for the most blatant, obvious cases of HTTP response splitting.
-If FindBugs found &lt;em&gt;any&lt;/em&gt;, you &lt;em&gt;almost certainly&lt;/em&gt; have more
-vulnerabilities that FindBugs doesn't report. If you are concerned about HTTP response splitting, you should seriously
+&lt;p&gt;SpotBugs looks only for the most blatant, obvious cases of HTTP response splitting.
```

Comment:
Spotbugs rebranding is in progress
These files are autogenerated by `BuildXmlFiles.groovy`